spec/requests/api/visioplainte/swagger_doc/rdvs_spec.rb

```diff
@@ -23,6 +23,32 @@ def self.rdv_response_schema
   end
 
   path "/api/visioplainte/rdvs" do
+    get "Lister les rdvs" do
+      with_visioplainte_authentication
+
+      tags "Rendez-vous"
+      description "Liste tous les rendez-vous, et permet de filtrer par date, par guichet, ou sur des rdv spécifiques"
+      parameter name: "from", in: :query, type: :string,
+                description: "datetime au format iso8601. Obligatoire sauf si le paramètre ids est utilisé.\
+                Permet de filtrer pour obtenir uniquement les rendez-vous qui commencent à partir de cette heure",
+                example: "2024-08-19T08:00:00+02:00", required: false
+      parameter name: "to", in: :query, type: :string,
+                description: "datetime au format iso8601.  Obligatoire sauf si le paramètre ids est utilisé.\
+                Permet de filtrer pour obtenir uniquement les rendez-vous qui commencent avant cette heure",
+                example: "2024-08-22T19:00:00+02:00", required: false
+      parameter name: "ids", in: :query, type: :array,
+                description: "Une liste d'ids des rendez-vous qu'on souhaite obtenir",
+                example: "123,456", required: false
+      parameter name: "guichet_ids", in: :query, type: :array,
+                description: "Une liste d'ids des guichets sur lesquels on veut filtrer les rendez-vous. Obligatoire si les paramètres from et to ne sont pas présents.",
+                example: "123,456", required: false
```

Comment:
et alors bizarrement j’ai un deuxième problème sur l’interface web swagger pour les guichet_ids uniquement, pas pour les ids.
quand j’arrive en faisant un hard refresh sur la page, je clique sur try it out je rentre ma clé d’api et les dates et je clique sur execute et en fait il passe en paramètre un `guichet_ids=123` qui semble venir de cet exemple mais qui est absolument invisible sur l’interface web. j’ai mis un moment à comprendre pourquoi je ne voyais aucun rdvs retournés par le endpoint.

peut-être qu’il faut passer un array ici à `example` ? ou alors simplement supprimer ces exemples pour éviter le pb

Reply:
oui, au final le plus simple c'est d'enlever les exemples. C'est ce que j'ai fait